gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -163,6 +161,89 @@ export default function NxGlobalSidebarPage() {
             </NxTable.Body>
           </NxTable>
         </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxGlobalSidebarFooter</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>showSupport</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  If true the support link is shown. At minimum <NxCode>supportLink</NxCode> below should be
+                  provided.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportIcon</NxTable.Cell>
```

Comment:
Do we have a requirement for this to be customizable?

Reply:
> We do not have a requirement that says that it is required, or customizable. I'm anticipating a need.

That seems premature to me.  A lot of the customizability you've allowed for in this component does.

Reply:
> > We do not have a requirement that says that it is required, or customizable. I'm anticipating a need.
> 
> That seems premature to me. A lot of the customizability you've allowed for in this component does.

@dturner32 care to weigh in?

Reply:
is this in reference to specific content in the footer?

Reply:
> is this in reference to specific content in the footer?

Yes, there is an icon next to the support text. The question is whether that's the only icon that can be used here in which case we'll hard code it or whether it is customizable.

Reply:
the support icon should not be custom and should always be the question mark

Reply:
> the support icon should not be custom and should always be the question mark

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/76bc776d69f2e7850cce375d90144cbe1b9ca8bb

Reply:
> Done here: 76bc776

Docs remain

Reply:
> > Done here: [76bc776](https://github.com/sonatype/sonatype-react-shared-components/commit/76bc776d69f2e7850cce375d90144cbe1b9ca8bb)
> 
> Docs remain

Docs removed: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/25c3048b360be327c90a2513c0db203d452a97e6